apps/ledger-live-desktop/static/i18n/el/app.json

```diff
@@ -1607,6 +1607,7 @@
         "notEnoughSpace": "Not enough storage",
         "supported": "Ledger Live supported",
         "not_supported": "Requires 3rd-party wallet",
+        "developedBy":"Developed by",
```

Comment:
New strings only need to be added to the english language, the rest is handled by the translations team